client.js

```diff
@@ -156,34 +391,131 @@ bot.on('message', function (message) {
+                            collector.on('collect', (reaction, user) => {
+                                if (user.bot) return;
+                                switch (reaction.emoji.name) {
+                                    case '💤': //alt: 🔕
+                                        // Snooze this watch for 6 hours
+                                        db.snooze('watch', watchId);
+                                        user.send(`Sleep is good.  Pausing notifications for the next 6 hours on your \`\`${item}\`\` watch on \`\`${server}\`\`.  Click 💤 again to unsnooze.  To snooze all watches, use \`\`!snooze\`\``).catch(console.error);
+                                        break;
+                                    case '❌':
+                                        // Delete this watch
+                                        db.endWatch(user.id, item, server);
+                                        user.send(`Got it! No longer watching auctions for ${item} on P1999 ${server} server.`);
+                                        break;
+                                    case '🔕':
+                                        // Ignore this seller's auctions for this watch
+                                        db.blockSeller(user.id, seller, null, watchId)
+                                        user.send(`Let's cut out the noise!  No longer notifying you about auctions from ${seller} with regard to this watch.\n  To block ${seller} on all present and future watches, use \`\`!add block: ${seller}`);
+                                        break;
+                                    case '♻': //extend watch
+                                        db.extendWatch(watchId)
+                                        user.send(`Good things come to those who wait.  I added another 7 days to your \`\`${formattedItem}\`\` watch.`);
+                                        break;
+                                    default:
+                                        break;
+                                }
+                            })
+                            collector.on('remove', (reaction, user) => {
+                                if (user.bot) return;
+                                switch (reaction.emoji.name) {
+                                    case '💤':
+                                        // unsnooze watch
+                                        db.unsnooze('watch', watch.id);
+                                        user.send(`Rise and grind.  No longer snoozing on your \`\`${item}\`\` watch on \`\`${server}\`\`.`).catch(console.error);
+                                        break;
+                                    case '❌':
+                                        // renew this watch
+                                        // db.endWatch(user.id, item, server);
+                                        user.send(`Got it! No longer watching auctions for ${item} on P1999 ${server} server.`);
```

Comment:
Wrong message, and doesn't do anything? This should be "now watching this auction again", and it should START a watch. You've just left "endWatch" commented...